packages/ddl-shim/src/upgrade.ts

```diff
@@ -698,14 +736,96 @@ class DDLUpgrade {
     writeDataviews(): DDL2.IView[] {
         const retVal: DDL2.IView[] = [];
         for (const id in this._ddl2Dataviews) {
+            console.group(id);
```

Comment:
@GordonSmith you should take a look at what this `console.group` output looks like. It makes this whole thing a lot easier to debug. BTW could we add a `console` wrapper to `common_Widget` so we can leave some highly meaningful console log statements inplace and enable/disable them at the application layer? I know we have `Logger`, but it would be tremendously helpful to be able to use something that behaves like the `console` object that we all know and love.